wlutil/config.py

```diff
@@ -1,6 +1,7 @@
 import glob
 import collections
 import json
```

Comment:
Is this import still necessary?

Reply:
nope. In fixing that I also found a bug in the exception handler for invalid yaml files. Anyway, both fixed now.